samples/custom_builder.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: experimental.kpack.pivotal.io/v1alpha1
+kind: CustomBuilder
+metadata:
+  name: my-custom-builder
+  namespace: default
+spec:
+  serviceAccount: default
+  tag: sample/custom-builder
+  stack:
+    baseBuilderImage: cloudfoundry/cnb:bionic
+  store:
+    image: cloudfoundry/cnb:bionic
```

Comment:
I wonder if we should not even have CustomBuilder samples until we stabilized the api and can point to released pack tooling.

Reply:
This was useful for smoke testing. We could move it to something like samples/experimental.